site/plugin-v2.md

```diff
@@ -0,0 +1,77 @@
+
+title: Integrating Docker via the Network Plugin (V2)
+menu_order: 65
+search_type: Documentation
+---
+
+ * [Installation](#installation)
+ * [Configuring the Plugin](#configuring)
+ * [Running Services or Containers Using the Plugin](#usage)
+
+Docker Engine version 1.12 introduced [a new plugin system (V2)](https://github.com/moby/moby/commit/f37117045c5398fd3dca8016ea8ca0cb47e7312b).
+This document describes how to use [the Network Plugin V2 of Weave Net](https://store.docker.com/plugins/weave-net-plugin).
+
+Before using the plugin, please keep in mind the plugin works only in Swarm
+mode.
+
+### <a name="installation"></a>Installation
+
+To install the plugin run the following command on _each_ host already
```

Comment:
Is it worth mentioning both masters and workers in addition to "on _each_ host"?